pages/styles/global.css

```diff
@@ -23,6 +23,10 @@ body {
 	background: #fff;
 }
 
+a:hover {
+    text-decoration: none;
+}
```

Comment:
This is one of the tags that should be within `global.css`. Not just the `:hover` selector but also for `:focus` and `:focus-visible`.

The color of the anchor tag should also reflect what was designed in the Figma prototype.